src/features/import/components/ImportDialog/Configure/Preview/index.tsx

```diff
@@ -149,6 +150,17 @@ const Preview = () => {
                     />
                   );
                 }
+
+                if (column.kind === ColumnKind.ENUM) {
+                  return (
+                    <EnumPreview
+                      key={columnIdx}
+                      currentSheet={currentSheet}
+                      fieldKey={column.field}
+                      fields={fields}
+                    />
+                  );
+                }
```

Comment:
Fixed

Reply:
Fixed in 9faecf999bbd3737a29ba3f0f0204f915158567a. There are still other problems but they seem to be related to the underlying mapping rather than just the preview.